public/src/components/admin/CourseCreationInfo.tsx

```diff
@@ -11,13 +11,13 @@ const CourseCreationInfo = (): JSX.Element => {
                 This is to keep the student roster for the different runs of the course separate.
             </p>
             <p>
-                <a className="badge-pill badge-success" href="https://github.com/enterprises/university-of-stavanger" target="_blank" rel="noopener noreferrer">
+                <a className="badge-pill badge-success" href={process.env.QUICKFEED_ENTERPRISE_URL ?? "https://github.com/organizations/plan"} target="_blank" rel="noopener noreferrer">
```

Comment:
See my revised `.env-template` file. Using a default value in the `.env` file avoids the hardcoded link here and the `??` thing.

Note: I renamed the var to `QUICKFEED_ORGANIZATION_URL` instead of enterprise.